src/Onboarding/OnboardingScreen.tsx

```diff
@@ -155,23 +170,55 @@ const PositionDots: FunctionComponent<PositionDotsProps> = ({
   )
 }
 
+const isIOS = Platform.OS === "ios"
+const skipButtonSpacingTop = isIOS ? Spacing.xSmall : Spacing.xxLarge
```

Comment:
Android handles `absolute` positioning differently from iOS in this particular circumstance. I think it's a bug. We need to do this to make the designs look the same.

Reply:
How do you feel about using a `Platform.select`? When we use one of the two platforms it kind of feels like we treat one as the default and the other as the special case(guilty of that 😄  )

Reply:
Totally, good call.